integration-tests/bats/json-diff.bats

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+
+    dolt sql <<SQL
+CREATE TABLE test (
+  pk BIGINT NOT NULL COMMENT 'tag:0',
+  c1 BIGINT COMMENT 'tag:1',
+  c2 BIGINT COMMENT 'tag:2',
+  c3 BIGINT COMMENT 'tag:3',
+  c4 BIGINT COMMENT 'tag:4',
+  c5 BIGINT COMMENT 'tag:5',
+  PRIMARY KEY (pk)
+);
+SQL
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+@test "json-diff: new row" {
+    dolt add .
+    dolt commit -m table
+    dolt sql -q 'insert into test values (0,0,0,0,0,0)'
+    dolt add .
+    dolt commit -m row
+
+    dolt diff -r json
```

Comment:
(minor) is this line doing something? Seems like the exit status check for the next line would capture any failure.

Reply:
It makes it so that you get the output the command you're testing when the test fails, kinda nice